src/pages/Dashboard/Families/FamilyDetail/FamilyDetail.tsx

```diff
@@ -1,26 +1,40 @@
 import { DashboardCard } from '@camp/components';
 import { messages } from '@camp/messages';
+import { useMatches } from '@camp/router';
+import { useEffect, useState } from 'react';
+
+import { useDraftFamilyDetailQuery } from '../../../../data-layer/operations/__generated__/typesAndHooks';
 
 export const FamilyDetail = () => {
   const t = messages.familyDetail.familyFields;
+  const [familyId, setFamilyId] = useState('');
+  const matches = useMatches();
+
+  useEffect(() => {
+    matches.map(match => setFamilyId(match.params.familyId));
+  }, [familyId, matches]);
+
+  const { data } = useDraftFamilyDetailQuery({
+    variables: { id: familyId },
+  });
 
-  return (
-    <DashboardCard
-      title={messages.familyDetail.title}
-      id={messages.familyDetail.id}
-    >
+  return data?.family?.__typename === 'DraftFamily' ? (
```

Comment:
don't we want to show the detail for `CompletedFamily` too ?

Reply:
Yeah, we can add it later.